goveralls.go

```diff
@@ -242,17 +243,23 @@ func process() error {
 	//
 	// Initialize Job
 	//
-	var jobId string
-	if travisJobId := os.Getenv("TRAVIS_JOB_ID"); travisJobId != "" {
-		jobId = travisJobId
-	} else if circleCiJobId := os.Getenv("CIRCLE_BUILD_NUM"); circleCiJobId != "" {
-		jobId = circleCiJobId
-	} else if appveyorJobId := os.Getenv("APPVEYOR_JOB_ID"); appveyorJobId != "" {
-		jobId = appveyorJobId
-	} else if semaphoreJobId := os.Getenv("SEMAPHORE_BUILD_NUMBER"); semaphoreJobId != "" {
-		jobId = semaphoreJobId
-	} else if jenkinsJobId := os.Getenv("BUILD_NUMBER"); jenkinsJobId != "" {
-		jobId = jenkinsJobId
+	jobId := *customJobId
+	if customJobId == nil || *customJobId == "" {
+		jobId = os.Getenv("CUSTOM_JOB_ID")
+	} 
+
+	if jobId == "" {
```

Comment:
It might make more sense to stick with existing style here. For example:

```go
var jobId string
if customJobId != nil {
	jobId = *customJobId
} else if customJobIdEnv := os.Getenv("CUSTOM_JOB_ID"); customJobIdEnv != "" {
	jobId = customJobIdEnv
} else if travisJobId := os.Getenv("TRAVIS_JOB_ID"); travisJobId != "" {
// ...

```